### DISCUSSION TITLE
Ability to show only differences

### DISCUSSION THREAD

**Discussion by brahmajitammana** [forcedotcom/salesforcedx-vscode#5759] [Ideas]
**Is your feature request related to a problem? Please describe.**
It is a possible enhancement. Ability to show only differences between the files would be quicker to see limited updates that are made.

**Comment by lcampos** [comment]
Hi @brahmaji83 thanks for opening this issue. Can you add a bit more detail to this ? Is this feedback specific to when you run the diff command, during conflict detection on source:push & source:pull or a new feature request ?

**Comment by scottlhenderson** [comment]
We only want to see actual file content changes.  I've settting in the git config of filemode = false but after retrieving source from manifest many files show as modified while diff shows no changes.
